### PR TITLE
Replace 'sponsor' with 'qualifying individual'

### PR DESCRIPTION
**Pull request by cvalarida**
Connected to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1781

**Update:** See [this comment](https://github.com/department-of-veterans-affairs/vets-website/pull/5146#issuecomment-289570391) for updated screenshots.

----

It's a bit...um...awkward in some places, I think, but I'll leave that to @goldenmeanie to iron out.

> image

> image

### CODE REVIEW

File: src/js/edu-benefits/5490/config/form.js
Comment (cvalarida) [inline src/js/edu-benefits/5490/config/form.js]: Should this change to something like `qualifying-individual/information`?

File: src/js/edu-benefits/5490/config/form.js
Comment (cvalarida) [inline src/js/edu-benefits/5490/config/form.js]: Same here. `qualifying-individual/service`?

**Comment by raquelromano-gov** [comment]
Oh wow...super weird.
@J-Quag @mollyblake Can either/both of you look at the above screenshots and verify that this is what they want?

**Comment by J-Quag** [comment]
Yeah this is what they wanted, but I agree this is super awkward @cvalarida  @raquelromano-gov 
  @goldenmeanie  Any thoughts on how we can change this?  Also maybe @amybehrens  can help? I can give you a run down on the context in person.

**Review by raquelromano-gov — APPROVED** [review]
@cvalarida Approving this so I don't block you, but I fear they didn't think this all the way through and they'll be surprised when they see all the changes. E.g., even the intro page instructions are bizarre, "The qualifying individual's Social Security number (required)" is sure to puzzle people.

**Comment by J-Quag** [comment]
**Intro Page:**
  put in qualifying individual's (Veteran's) Social Security Number

**Benefit History:** 
  This drop down, if you select that box, can be Sponsor. The only reason why Sponsor was not wanted for the form itself is that the granting of a benefit was not necessarily a choice of the Veteran. That drop down box also should be differentiated between the information entered later on even though they could potentially be the same.

**Sponsor Service**
header should be:  
Qualifying Individual's Service Information

Branch of Service
Are they currently on Active Duty?
Do they have an outstanding felony and/or warrant?

Also Sponsor Information:
qi-5490-edu

**Comment by raquelromano-gov** [comment]
LMK if/when you understood the conclusion and want me to take another look at this. I definitely did not follow what the decision was after today's call.

**Comment by J-Quag** [comment]
@cvalarida caught my mistake in the content.

**Comment by cvalarida** [comment]
Updated screenshots:

## Intro page
> image

## Benefit history
> image

> image

## Sponsor information
> image

## Sponsor service
> image

**Review by raquelromano-gov — COMMENTED** [review]
Want me to take another look? I approved this long ago when we actually thought they wanted QI all over the place.

**Comment by cvalarida** [comment]
@raquelromano-gov Sure, just really quickly. Mostly I'm waiting for @goldenmeanie's :+1:. From what I understand, he was meeting with somebody to talk about this among other things and come to a final decision.

**Comment by goldenmeanie** [comment]
We were focused on the disclaimer stuff the other day. I will run through this with Shay 3/31.

**Comment by J-Quag** [comment]
Shay won't be in. Let's go over it and make a decision to move forward with.

**Review by goldenmeanie — COMMENTED** [review]
We've convinced stakeholders to stick with Sponsor. Abandoning this!

**Comment by cvalarida** [comment]
Phew!

**Pull request closed without merging**
